### PR TITLE
monitor prometheus

### PR DESCRIPTION
**Pull request by TheoBrigitte**
Towards: https://github.com/giantswarm/giantswarm/issues/19310

### CODE REVIEW

File: helm/prometheus-operator-app/templates/prometheus/service.yaml [hunk 1]
```diff
@@ -6,6 +6,7 @@ metadata:
   namespace: {{ template "kube-prometheus-stack.namespace" . }}
   labels:
     app: {{ template "kube-prometheus-stack.name" . }}-prometheus
+    giantswarm.io/monitoring: "true"
```
Comment (ubergesundheit) [inline helm/prometheus-operator-app/templates/prometheus/service.yaml:9]: What about also adding the annotation `giantswarm.io/monitoring-app-label: {{ .Chart.Name }}`

See https://intranet.giantswarm.io/docs/monitoring/monitoring_of_apps/#service-object-specification

**Comment by TheoBrigitte** [comment]
Tested and working :heavy_check_mark: 

image
image

File: helm/prometheus-operator-app/templates/prometheus/service.yaml (hunk 1 above)
Comment (TheoBrigitte) [reply helm/prometheus-operator-app/templates/prometheus/service.yaml:9]: Currently the `app` shows was `app="prometheus-operator-app-prometheus"`, do you feel like this is not enough ?

Comment (ubergesundheit) [reply helm/prometheus-operator-app/templates/prometheus/service.yaml:9]: The annotation makes the `app` label in prometheus stable. If an user changes the name the `app` label will change when its left like this.

**Pull request merged**